### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by andreafioraldi**
We should merge, the users should benefit of the calibration fix ASAP.

### CODE REVIEW

**Comment by vanhauser-thc** [comment]
start with the review guys, I already did mine :) (and pushed the fixes)

**Comment by andreafioraldi** [comment]
I re-checked better the problem and we should never let a queue entry have exec_cksum = 0.

**Comment by vanhauser-thc** [comment]
now that we have 64bit checksums the chances are 2^64 that this happens.
we could do a `if (!chksum) return 1; else return chksum:` in hash64 though if you would prefer that.

File: afl-plot [hunk 1]
```diff
@@ -31,40 +35,40 @@ an empty directory where this tool can write the resulting plots to.
...
+#  T2="$?"
+#
+#  if [ "$T1" = "0" -o "$T2" = "0" ]; then
+#
+#    echo "[-] Error: this script shouldn't be used with shared /tmp directories." 1>&2
+#    exit 1
+#
+#  fi
+#
+#fi
```
[the remark this answers is not in the document]
Comment (vanhauser-thc) [reply afl-plot:62]: as you can see it is commented out :)

**Comment by vanhauser-thc** [comment]
ok found and fixed the bug. autodict in the forkserver had an issue. compile opt for helper object files is set back to -O3

**Comment by vanhauser-thc** [comment]
for me it is fine to merge now.
@domenukk you want to rename afl_llvm_whitelist before or after the merge?

**Pull request merged**